### PR TITLE
Add rbac and psp

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards [#2296](https://github.com/giantswarm/giantswarm/issues/2296)

I have added a service account and a role with the k8s methods I have seen our code uses.

The role is at the namespace level because I understand draughtsman only reads config map in the same space it is and applies the changes. I have seen is using the helm installed client to apply the changes, but how do we install the helm client in the host?
Right now all system authenticated users have PSP restricted. So I do think any special PSP should be applied to draughtsman, but helm client which is the one could need more privileges, WDYT?

cc/ @puja108

### CODE REVIEW

File: helm/draughtsman-chart/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+kind: Role
+metadata:
+  name: draughtsman
+  namespace: draughtsman
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmap
+      - secrets
```
Comment (puja108) [inline helm/draughtsman-chart/templates/rbac.yaml]: I think we can limit cm and secret to a single named one, as it's always the same

Comment (pipo02mix) [reply helm/draughtsman-chart/templates/rbac.yaml]: yes it has sense

**Comment by pipo02mix** [comment]
@puja could you review it again?

**Comment by xh3b4sd** [comment]
Note that when we make changes in draughtsman we should also sync those changes to draughtsman-operator as well. I still hope to use it soon. Especially now that we sorted the goroutine leak issues. This was the reason we postponed using the draughtsman-operator.

**Comment by pipo02mix** [comment]
@xh3b4sd Could you explain the differnet between those? I am not sure if they will need same RBAC rules

**Comment by xh3b4sd** [comment]
The operator is doing the same. It is only implemented with the operator design. When your work is about RBAC I assume it is because of K8s access. Drauhtsman and draughtsman-operator both access k8s. Note that there is also the draughtsman-eventer because of the operator design.

**Pull request merged**